backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/process/ProcessExecutionResult.java

```diff
@@ -885,7 +884,7 @@ private RecordsToOpen(
 			this.records = ImmutableList.copyOf(records);
 			this.windowIdString = StringUtils.trimBlankToNull(adWindowId);
 			this.target = target != null ? target : OpenTarget.GridView;
-			this.targetTab = targetTab != null ? targetTab : TargetTab.SAME_TAB;
+			this.targetTab = targetTab != null ? targetTab : target == OpenTarget.GridView ? TargetTab.SAME_TAB_OVERLAY : TargetTab.SAME_TAB;
```

Comment:
avoid having logic here.
pls decide what kind of targetTab we need on the caller method,.

Reply:
@teosarca then I would leave it as default in the builder to SAME_TAB unless set different and make sure whenever opentarget.gridview is used the targetTab gets passed as targettab.same_tab_overlay. WDYT?